src/components/eRACommons.js

```diff
@@ -126,7 +126,7 @@ export const eRACommons = hh(class eRACommons extends React.Component {
 
     return (
       div({ className: this.props.className }, [
-        label({ className: 'control-label' }, ['NIH eRA Commons ID*']),
+        label({ className: 'control-label' }, ['NIH eRA Commons ID ', span({ className: 'italic display-inline' }, ['(Required for DARs)'])]),
```

Comment:
We should revert this change for now. The message displayed is true, but this component is also displayed on the DAR application page so adding this text here makes it slightly more confusing there since you're already making a DAR. 

What we should do, in a separate story/PR, is to update this component to take a title as a prop. The places that use this can then provide whatever title text that makes sense for their usage.

Lastly, the term `DAR` is something we use internally to refer to a `Data Access Request`, so using that term here would be confusing to some users, especially international ones.